### DISCUSSION TITLE
Multi User

### DISCUSSION THREAD

**Discussion by spacefrogfeds** [Forceu/Gokapi#218] [Q&A]
Hi, What I would to do is for different users to not be able to easily see what other users have been recently uploading. 
I know gokapi doesn't have a multi-user function. Can it be done with authentik, authelia etc? I've been playing with authentik & header authentication and I've only been able to achieve an additional login screen. So everyone has to log in twice for the same result.

**Comment by Forceu** [comment]
This has been a requested feature (#61), but unfortunately requires a lot of changes. It is on the todo list however and most likely one of the next features

**Reply by spacefrogfeds** [reply]
Thanks for the quick reply. I understand that it is far beyond your original scope and appreciate all your efforts.

**Reply by Forceu** [reply]
This has been added in v2.0, I therefore close this issue.
